packages/core/src/common/console-logger-server.ts

```diff
@@ -7,25 +7,42 @@
 
 import { injectable } from "inversify";
 import { LogLevel } from './logger';
-import { ILoggerServer, ILoggerClient } from './logger-protocol';
+import { ILoggerServer } from './logger-protocol';
+
+const originalConsoleLog = console.log;
 
 @injectable()
 export class ConsoleLoggerServer implements ILoggerServer {
-    setLogLevel(name: string, logLevel: number): Promise<void> {
-        return Promise.resolve();
+
+    async setLogLevel(): Promise<void> {
+        // NOOP
```

Comment:
Setting the log level could be supported, no?

Reply:
Can be. It was not. I will add.